README.md

```diff
@@ -369,6 +369,11 @@ using the name of the corresponding environment variable.
 
 * `CLVK_CONFIG_FILE` specifies the path to an additional configuration file.
 
+* `CLVK_IGNORE_OUT_OF_ORDER_EXECUTION` can be used to ignore out of order execution checks
+
+   * 0: does not ignore OOOE checks and validates if hardware can support it (default)
+   * 1: ignores the aforementioned checks
```

Comment:
```suggestion
   * 0: creating an out-of-order queue results in a failure (default)
   * 1: creating an out-of-order queue is supported but it will function as an in-order queue
```